Configurations/50-nonstop.conf

```diff
@@ -168,13 +168,13 @@
     'nonstop-model-put' => {
```

Comment:
Do the model names also need to be swapped?

Reply:
> Do the model names also need to be swapped?

The model names are fine. FLOSS is not needed for PUT. However, I'm struggling with the SPT compile.